scss/forms/_star-rating.scss

```diff
@@ -9,6 +9,7 @@
   --#{$prefix}star-rating-checked-icon: #{$form-star-rating-checked-icon};
   --#{$prefix}star-rating-unchecked-icon: #{$form-star-rating-unchecked-icon};
 
+  display: flex;
```

Comment:
Both the `display: flex` and the removal of `float: left` changes were needed otherwise, the focus wasn't handled correctly. Here is the rendering without the changes: 
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/91960143/6e4c06ee-092e-40c5-bab6-d30323d1612f)